webview-ui/src/i18n/locales/en/settings.json

```diff
@@ -396,6 +396,10 @@
 	},
 	"experimental": {
 		"warning": "⚠️",
+		"AUTO_CONDENSE_CONTEXT": {
+			"name": "Automatically condense the context window",
```

Comment:
Maybe "Intelligently condense the context window"? Might be good to somehow compare this to the sliding window that we currently use.